### PR TITLE
VATRP-1903: Sanity check and crash fix for legal release

### PR DESCRIPTION
**Pull request by zackm0571**

### CODE REVIEW

File: src/org/linphone/ContactsManager.java [hunk 1]
```diff
@@ -565,7 +563,7 @@ public void run() {
 							continue;
 
 						//Remove linphone contact tag if the contact has no sip address
-						if (LinphoneActivity.isInstanciated() && LinphoneActivity.instance().getResources().getBoolean(R.bool.use_linphone_tag)) {
```
Comment (zackm0571) [inline src/org/linphone/ContactsManager.java:568]: Sanity check. This boolean will always be true and is tied to linphone implementation

File: src/org/linphone/DialerFragment.java [hunk 2]
```diff
@@ -478,10 +478,10 @@ else if (o > 180 - sensativity && o < 180 + sensativity)//when o is around 180 w
 			else if (o > 270 - sensativity && o < 270 + sensativity)//when o is around 180 we set the degrees to 180
 				degrees = 270;
 
-			Log.d("onOrientationChanged_Dialer", o);
+			/*Log.d("onOrientationChanged_Dialer", o);
 			Log.d("degrees", degrees);
 			Log.d("mAlwaysChangingPhoneAngle", mAlwaysChangingPhoneAngle);
-
+			*/
```
Comment (zackm0571) [inline src/org/linphone/DialerFragment.java:484]: Removing verbose logs cluttering up adb

File: src/org/linphone/LinphoneActivity.java [hunk 3]
```diff
@@ -1255,13 +1255,6 @@ protected void onPause() {
 	@Override
 	protected void onResume() {
 		super.onResume();
-		final SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(ctx);
-		boolean hasAcceptedRelease = prefs.getBoolean("accepted_legal_release", false);
-		if(!hasAcceptedRelease){
```
Comment (zackm0571) [inline src/org/linphone/LinphoneActivity.java:1260]: Relocate this logic to launcher activity

File: src/org/linphone/setup/GenericLoginFragment.java [hunk 4]
```diff
@@ -145,7 +138,6 @@ public void onClick(View v) {
 	@Override
 	public void onClick(View v) {
 		int id = v.getId();
-		
 		if (id == R.id.btn_prv_login) {
 			if (login.getText() == null || login.length() == 0 || password.getText() == null || password.length() == 0 || domain.getText() == null || domain.length() == 0) {
 				Toast.makeText(getActivity(), getString(R.string.first_launch_no_login_password), Toast.LENGTH_LONG).show();
```
Comment (zackm0571) [inline src/org/linphone/setup/GenericLoginFragment.java:143]: Clean up imports

**Comment by patrickmwatson** [comment]
+1 LGTM

**Pull request merged**